src/test/java/org/junit/tests/experimental/AssumptionTest.java

```diff
@@ -188,9 +188,18 @@ public void failingAssumptionInConstructorIgnoresClass() {
         assertThat(testResult(AssumptionFailureInConstructor.class), isSuccessful());
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void assumeWithExpectedException() {
-        assumeTrue(false);
+    public static class TestClassWithAssumptionFailure {
+
+        @Test(expected = IllegalArgumentException.class)
```

Comment:
I know it was there before, but the `expected` parameter is confusing and should be removed.